docs/writing-tests/test-runner-with-vitest.mdx

```diff
@@ -0,0 +1,502 @@
+
+<If renderer="react">
+  If you're using Next.js, use `@storybook/experimental-nextjs-vite/vite-plugin`:
+
+  ```js title="vitest.config.ts"
+  import { defineConfig, mergeConfig } from 'vitest/config';
+  import { storybookTest } from '@storybook/experimental-addon-test/vite-plugin';
+  import { storybookNextjsPlugin } from '@storybook/experimental-nextjs-vite/vite-plugin';
+
+  import viteConfig from './vite.config';
+
+  export default mergeConfig(
+    viteConfig,
+    defineConfig({
+      plugins: [
+        storybookTest(),
+        storybookNextjsPlugin(), // 👈 Use the plugin here
+      ],
+      // ...
+    })
+  );
+  ```
+</If>
+
+<If renderer="vue">
+  Vue projects should use `@storybook/vue3-vite/vite-plugin`:
+
+  ```js title="vitest.config.ts"
+  import { defineConfig, mergeConfig } from 'vitest/config';
+  import { storybookTest } from '@storybook/experimental-addon-test/vite-plugin';
+  import { storybookVuePlugin } from '@storybook/vue3-vite/vite-plugin';
+
+  import viteConfig from './vite.config'
+
+  export default mergeConfig(
+    viteConfig,
+    defineConfig({
+      plugins: [
+        storybookTest(),
+        storybookVuePlugin(), // 👈 Use the plugin here
+      ],
+      // ...
+    })
+  );
+  ```
+</If>
+
+<If renderer="svelte">
+  If you're using SvelteKit, use `@storybook/sveltekit/vite-plugin`:
+
+  ```js title="vitest.config.ts"
+  import { defineConfig, mergeConfig } from 'vitest/config';
+  import { storybookTest } from '@storybook/experimental-addon-test/vite-plugin';
+  import { storybookSveltekitPlugin } from '@storybook/sveltekit/vite-plugin';
+
+  import viteConfig from './vite.config';
+
+  export default mergeConfig(
+    viteConfig,
+    defineConfig({
+      plugins: [
+        storybookTest(),
+        storybookSveltekitPlugin(), // 👈 Use the plugin here
+      ],
+      // ...
+    })
+  );
+  ```
+</If>
+
+The above example uses the framework's plugin in a Vitest configuration file. You can also use it in a Vitest workspace file, if that is how your project is configured.
+
+### Example configuration files
+
+When the addon is set up automatically, it will create or adjust your Vitest configuration files for you. If you're setting up manually, you can use the following examples as a reference when configuring your project.
+
+<details>
+  <summary>Example Vitest setup file</summary>
+
+ Storybook stories contain configuration defined in `.storybook/preview.js|ts`. To ensure that configuration is available to your tests, you can apply it in a Vitest setup file. Here's an example of how to do that:
+
+```ts title=".storybook/vitest.setup.ts"
+import { beforeAll } from 'vitest';
+// Replace your-renderer with the renderer you are using (e.g., react, vue3, svelte, etc.)
+import { setProjectAnnotations } from '@storybook/your-renderer';
+
+import * as projectAnnotations from './preview';
+
+const project = setProjectAnnotations(projectAnnotations);
```

Comment:
it's easier if it's an array already so that users know they can add more. I made this change to the code generated by the addon as well
```suggestion
const project = setProjectAnnotations([projectAnnotations]);
```